common/app/views/fragments/atoms/snippets/timeline.scala.html

```diff
@@ -28,9 +14,9 @@
       <div class="explainer-snippet__item">
         <time class="explainer-snippet__event-date" datetime="">
```

Comment:
Semantically, each date should be its own `time` element. If you move the logic out from the template, let's go all the way have do:
```
<time class="explainer-snippet__event-date" datetime="@item.date.toMachineReadable">
  @item.data.toHumanReadable(item.dateFormat)
</time>@item.toDate.map { toDate =>–<time class="explainer-snippet__event-date" datetime="@toDate.toMachineReadable">@toDate.toHumanReadable(item.dateFormat)</item>
}